tests/MongoDB.Driver.Core.Tests/Core/Operations/EndTransactionOperationTests.cs

```diff
@@ -30,17 +30,20 @@ public class EndTransactionOperationTests
         public void constructor_should_initialize_instance()
         {
             var writeConcern = new WriteConcern();
+            var recoveryToken = BsonDocument.Parse("{section: 31}");
```

Comment:
Let's not overuse parsing for simple documents. This could just as easily be:

    var recoveryToken = new BsonDocument("section", 31);

Reply:
Fixed